src/components/map/layers/earthEngine/styles/EarthEnginePopup.module.css

```diff
@@ -30,36 +32,46 @@
     font-weight: normal;
 }
 
-.table thead th {
-    text-align: left;
-    padding-bottom: var(--spacers-dp4);
+.table thead {
+    border-bottom: 1px solid #eee;
 }
```

Comment:
I suggest styling the `thead` with `color: var(--colors-grey600); font-weight: 500;`